### PR TITLE
gems

### PR DESCRIPTION
**Pull request by matoni109**
initial gems setup

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,172 @@
...
+puts "done cleaning house.."
+
+puts "-- Making Humans"
+## kill all images in all models ##
+
+## User Master OverLoad # HOLD GME !! ##
+1.times do
+  file = File.open("./db/avatars/00.jpeg")
+
+  make_me = User.create!(
```
Comment (julianharr) [inline db/seeds.rb:31]: Let's work on Seed file collectively

File: db/seeds.rb [hunk 2]
```diff
@@ -1,7 +1,172 @@
...
+count = 0
+
+11.times do
+  file = File.open("./db/avatars/#{count}.jpeg")
+
+  ## make the instance
+  make_me = User.create!(
+    first_name: Faker::Name.first_name,
+    last_name: Faker::Name.last_name,
+    bio: Faker::TvShows::Buffy.quote,
```
Comment (julianharr) [inline db/seeds.rb:69]: 🚫  Buffy quotes 😂

Comment (matoni109) [reply db/seeds.rb:69]: yeah i"m going to pull it all from the git hub api but I'll sort that out tonight 
we need to migrate out the models also for the green features

**Comment by julianharr** [comment]
Awesome work bro - friendship gem looks amazing. Good find. Going to park this merge until we're working together. Just want to make sure we're all on the same page with the schema and want us all to work on the seed file 👍

**Comment by matoni109** [comment]
yep thats cool
I've got it to a start point => we'll rip everything from GITHUB for users so the info is real

**Pull request merged**